lib/hanami/environment.rb

```diff
@@ -132,9 +138,8 @@ class Environment
     # located under the config directory. All the settings in those files will
     # be exported as `ENV` variables.
     #
-    # Master .env file is ignored to suggest clear separation of environment
-    # configurations and discourage putting sensitive information into source
-    # control.
+    # This table: https://github.com/bkeepers/dotenv#what-other-env-files-can-i-use
```

Comment:
I'm not sure the nest place to put this info but the table on this page is very useful so I think it's good to include it somewhere.

Reply:
Addressed here: https://github.com/hanami/hanami/pull/1004/commits/6dcc93417abc81f6d6bf2de729d0581dfb50392a